tests/test_learning_k_means.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for landmarks space.
+"""
+
+import geomstats
+import geomstats.backend as gs
+import geomstats.tests
+
+from geomstats.geometry import hyperbolic_space
```

Comment:
```from geomstats.geometry.hyperbolic_space import HyperbolicSpace```
and same for hypersphere.

It's the syntax we have used elsewhere, it's so that the code is shorter when we instantiate a space or a metric